Khan/vite
# FEI-4306: Handle websocket errors and log them in the server

## Summary:
Vite's websocket server code wasn't adding an .on('error', fn) handler on the socket that was created.  This resulted in vite crashing whenever there was an error on the socket.  While we were able fix some of the errors by changing the client code to close the socket before the page unloads, there are other situations where the socket connection is broken that can't be handled on the client.  One such case is when the machine goes to sleep which also terminates the socket.

This PR fixes the crash by adding an .on('error', fn) handler.  The handler simply logs the error.

Issue: FEI-4306

## Test plan:
In webapp:
- apply the same change to node_modules/vite/dist/node/chunks/dep-546f6c2b.js in webapp (minus the TypeScript types)
- `make run-vite`
- in a separate terminal `make run-viteproxy`
- load localhost:8224
- put the machine to sleep
- wake it up
- see that an error was logged in the run-vite terminal window, but that it's still running
- navigate to the login page to see that `vite` is still working correctly

<img width="1075" alt="Screen Shot 2022-03-04 at 10 30 18 AM" src="https://user-images.githubusercontent.com/1044413/156792011-f0ee1a77-c84e-4542-bd24-1c44e7bf6bd0.png">
